cmd/storj/cmd/cp.go

```diff
@@ -35,76 +37,141 @@ func init() {
 func copy(cmd *cobra.Command, args []string) (err error) {
```

Comment:
This method is quite long. How about extracting the three different copy cases in different helper mehtods?

Reply:
Done